pyblish/plugin.py

```diff
@@ -780,6 +782,46 @@ def current_host():
     return pyblish._registered_hosts[-1] or "unknown"
 
 
+def register_callback(signal, callback):
+
+    if signal in pyblish._registered_callbacks:
+        pyblish._registered_callbacks[signal].append(callback)
+    else:
+        pyblish._registered_callbacks[signal] = [callback]
+
+
+def deregister_callback(callback):
```

Comment:
Hm, there's probably no need for both `deregister_callback` and `deregister_callbacks`. Sure, it will reduce typing, but increase maintenance. Remember, we will need to keep both of these functions forever and ever. The less of that the better!

Additionally, I would make the signature identical to the registration. Just so we don't need to remember two signatures.

```python
deregister_callback("myEvent", my_callback)
```